plugins/swiot/src/swiotplugin.cpp

```diff
@@ -343,6 +343,7 @@ void SWIOTPlugin::startTutorial() {
 		auto configTool = configTme->tool();
 		auto parent = Util::findContainingWindow(configTool);
 		auto tut = new gui::TutorialBuilder(configTool, ":/swiot/tutorial_chapters.json", "config", parent);
+		tut->setTitle("CONFIG ");
```

Comment:
Do you need the space in "CONFIG " ? If yes, it could suggest that a style change is needed to acomodate spacing.